opus/application/test_api/api_test_helper.py

```diff
@@ -166,3 +167,25 @@ def _run_csv_equal_file(self, url, exp_file):
         print('Expected:')
         print(expected)
         self.assertEqual(resp, expected)
+
+    def _run_zip_equal(self, url, expected):
+        print(url)
+        response = self._get_response(url)
+        self.assertEqual(response.status_code, 200)
+        if '__cart/download.json' in url:
+            jdata = json.loads(response.content)
+            file = jdata['filename']
+            filename = file[file.rindex('/'):]
```

Comment:
Since we aren't guaranteed what the user has set for TAR_FILE_URL_PATH, how about just stripping off TAR_FILE_URL_PATH (e.g. replace(TAR_FILE_URL_PATH, '') instead of looking for the last "/"?